iphone/Maps/UI/Settings/MWMSettingsViewController.mm

```diff
@@ -150,6 +164,23 @@ - (void)configCommonSection {
                                              isOn:[MWMSettings compassCalibrationEnabled]];
 }
 
+- (void)show3dBuildingsAlert:(UITapGestureRecognizer *)recognizer
+{
+  UIAlertController *alert =
+  [UIAlertController alertControllerWithTitle:L(@"pref_map_3d_buildings_title")
+                                      message:L(@"pref_map_3d_buildings_disabled_summary")
+                               preferredStyle:UIAlertControllerStyleAlert];
+
+  UIAlertAction* yesButton = [UIAlertAction actionWithTitle:@"OK"
+                                                      style:UIAlertActionStyleDefault
+                                                    handler:^(UIAlertAction * action) {
+      [alert dismissViewControllerAnimated:YES completion:nil];
```

Comment:
Is this line really required? Won't it work with nil or empty handler?
(I'm thinking about the lifetime or alert controller in such cases, will it be ever freed?)

Reply:
You are right. It works with `handler:nil`.